plugin/evm/wrapped_block.go

```diff
@@ -172,7 +189,21 @@ func (b *wrappedBlock) handlePrecompileAccept(rules extras.Rules) error {
 
 // Reject implements the snowman.Block interface
 // If [b] contains an atomic transaction, attempt to re-issue it
-func (b *wrappedBlock) Reject(context.Context) error {
+func (b *wrappedBlock) Reject(_ context.Context) error {
```

Comment:
You change this a lot. I think we prefer no name rather than `_`
```suggestion
func (b *wrappedBlock) Reject(context.Context) error {
```